developer-docs-site/docs/concepts/basics-gas-txn-fee.md

```diff
@@ -2,71 +2,122 @@
+To conduct any transaction with the Aptos blockchain on the mainnet, you are required to pay a processing fees. This includes transactions from a client application, or from a stake owner, a node operator or a voter. Think of this processing fee in the following terms: 
 
-When a client submits a transaction for execution to the Aptos blockchain, it contains a specified:
+Any blockchain, for example, the Aptos blockchain, exists in a distributed network of computing and storage resources. The Aptos mainnet is one such distributed network, holding the Aptos blockchain. 
 
-* `max_gas_amount`: The maximum amount of gas units that can be used to execute the transaction. This bounds the computational resources that can be consumed by the transaction.
-* `gas_price`: The gas price in the blockchain's utility token. Gas price is a way to translate from gas units (the abstract units of resources consumed by the virtual machine) to a transaction fee in the blockchain's utility token.
+The processing fee you are required to pay is based on the computing and storage resources in the Aptos mainnet to:
 
-The transaction fee charged to the client will be at most `gas_price * max_gas_amount`.
+1. Process your transactions on the blockchain.
+2. Propagate the validated record throughout the distributed network of the mainnet, and
+3. Store the validated record in the distributed blockchain storage.  
 
-The gas price, and hence the transaction fee, should follow the market characteristics of the Aptos blockchain as resource supply and demand fluctuate.
+:::tip 
+Conceptually, this fee can be thought of as quite similar to how we pay for our home electricity or water utility.
+:::
+ 
+### Prioritizing your transaction
 
-## Types of resource usage
+You can even bump up your transaction to a higher priority level on the blockchain by paying a larger processing fee. In your transaction, you can commit to paying a gas price that is higher than the market gas price. This is one way you can move your transaction higher in the priority list and have it  processed quicker. 
 
-For the virtual machine (VM) to execute a transaction, the gas system needs to track the primary resources used by the transaction. These fall into three resource dimensions:
+### Unit of gas
 
-1. The computational cost of executing the transaction.
-2. The network cost of propagating the transaction through the Aptos ecosystem.
-3. The storage cost of data created and used during transaction execution on the Aptos blockchain.
+You can make a simple transaction, or a complicated transaction that requires the blockchain to perform lots of computation and distributed storage. In either case, you will be required to spend a processing fee sufficient to complete the transaction. This is where the notion of **gas** becomes clear**.** Here is how it works:
```

Comment:
I appreciate the boldness, but I'm not sure a period needs to be bolded?

Reply:
Haha, I like the pun. That wasn't intended.